apps/website/src/examples/FileUpload.main.tsx

```diff
@@ -3,20 +3,19 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 import * as React from 'react';
-import { FileUpload, FileUploadTemplate } from '@itwin/itwinui-react';
+import { FileUpload, FileUploadCard, Box } from '@itwin/itwinui-react';
 
 export default () => {
-  const [files, setFiles] = React.useState([]);
-
+  const [files, setFiles] = React.useState<Array<File>>([]);
   return (
     <FileUpload
+      style={{ position: 'absolute' }}
       onFileDropped={(files) => {
-        setFiles(Array.from(files));
+        setFiles(files);
+        action(`${files.length} files uploaded`)();
```

Comment:
I don't think action is valid in website example.

Reply:
Right. I removed action